### DISCUSSION TITLE
(Feature request) use bucanero's encryption method for ps2iso on HEN?

### DISCUSSION THREAD

**Discussion by ghost** [aldostools/webMAN-MOD#1143] [Ideas]
Also I will test it if you send me a test build so that I can install DBHFW for HEN. If @bucanero approves of it.

**Comment by ghost** [comment]
My example is like if either select button and encrypt iso or just select a non encrypted iso and it will say it's encrypting please wait. While deleting the original ISO that you dumped.

**Comment by aldostools** [comment]
For PS2 encryption use multiMAN or Apollo Save Tool. Adding the encryption algorithm to the plugin would be too heavy and would consume too much of the limited amount of memory available in XMB.
